SignalServiceKit/src/Storage/OWSOrphanedDataCleaner.m

```diff
@@ -83,20 +75,18 @@ + (void)auditAndCleanup:(BOOL)shouldCleanup completion:(void (^_Nullable)(void))
                                               }];
     }];
 
-    CleanupLogDebug(@"%@ fileCount: %lu", self.logTag, (unsigned long)fileCount);
-    CleanupLogDebug(@"%@ totalFileSize: %lld", self.logTag, totalFileSize);
-    CleanupLogDebug(@"%@ attachmentStreams: %d", self.logTag, attachmentStreamCount);
-    CleanupLogDebug(
-        @"%@ attachmentStreams with file paths: %lu", self.logTag, (unsigned long)attachmentFilePaths.count);
+    DDLogDebug(@"%@ fileCount: %lu", self.logTag, (unsigned long)fileCount);
+    DDLogDebug(@"%@ totalFileSize: %lld", self.logTag, totalFileSize);
+    DDLogDebug(@"%@ attachmentStreams: %d", self.logTag, attachmentStreamCount);
+    DDLogDebug(@"%@ attachmentStreams with file paths: %lu", self.logTag, (unsigned long)attachmentFilePaths.count);
```

Comment:
This logging won't work in tests now, right?

Reply:
From what I can tell, it should work fine. The Signal-iOS test suite is logging without issue.